docs/INSTALL.md

```diff
@@ -0,0 +1,83 @@
+## Prerequisite
+If you are using kubernetes distribution with no running OLM you can use [operator-sdk] cli
+```shell
+$ operator-sdk olm install
+```
+
+## Installation
+To get a functioning remediation mechanism you can go to the operator page on [operator hub]
+
+Alternatively you can run`$ kubectl create -f https://operatorhub.io/install/node-healthcheck-operator.yaml`
+
+By default, OLM will resolve Poison-Pill as a dependency and will install the latest available version
+in the current catalog or other catalog with higher priority.
+
+## Post install
+
+>NOTE: OLM operators goes under `operators` namespace by default.
+>      The Poison pill config and other resource in this example goes under `default` but 
+>      can go under every desired namespace.
+
+- create `PoisonPillConfig` object ([ongoing work to make it automatic][ppil-auto-config])
+```shell
+cat << EOF | kubectl create -f -
+apiVersion: poison-pill.medik8s.io/v1alpha1
+kind: PoisonPillConfig
+namespace: default
+metadata:
+  name: poison-pill-config
+spec: {}
+EOF
+```
+- create a poison pill remediation template
+```shell
+cat << EOF | kubectl create -f -
+apiVersion: poison-pill.medik8s.io/v1alpha1
+kind: PoisonPillRemediationTemplate
+namespace: default
+metadata:
+  name: ppill-template
+spec: {}
+```
+
+- create Node-Healthcheck CR that points to poison-pill remediation
+```shell
+cat << EOF | kubectl create -f -
+apiVersion: remediation.medik8s.io/v1alpha1
+kind: NodeHealthCheck
+namespace: default
+metadata:
+  name: nodehealthcheck-sample
+spec:
+  unhealthyConditions:
+    - type: Ready
+      status: Unknown
+      duration: 300s
+  remediationTemplate:
+    kind: PoisonPillRemediationTemplate
+    apiVersion: poison-pill.medik8s.io/v1alpha1
+    name: ppill-template
+    namespace: default
+```
+
+Your cluster should have 2 deployments and 1 daemonset:
+
+```shell
+$ kubectl get deploy -n operators
+NAME                                           READY   UP-TO-DATE   AVAILABLE   AGE
+node-healthcheck-operator-controller-manager   1/1     1            1           3m8s
+poison-pill-controller-manager                 1/1     1            1           3m10s
+
+### TODO fix rbac error that preventing the ds from rolling
```

Comment:
which rbac issue? I'm not familiar with one

Reply:
came up locally. not sure why yet

Reply:
make sure the ppc namespace equals poison pill operator namespaces